app/controllers/AccessibilityController.scala

```diff
@@ -93,30 +93,30 @@ class AccessibilityController @Inject() (
   accessibilityProblemConfirmationPage: AccessibilityProblemConfirmationPage,
   errorPage: InternalErrorPage,
   headerRetriever: RefererHeaderRetriever
-)(implicit val appConfig: AppConfig, val executionContext: ExecutionContext)
+)(using AppConfig, ExecutionContext)
     extends FrontendController(mcc)
     with DeskproSubmission
     with I18nSupport {
 
-  implicit def lang(implicit request: Request[_]): Lang = request.lang
+  given lang(using request: Request[?]): Lang = request.lang
 
   def index(
     service: Option[String],
     userAction: Option[String],
     referrerUrl: Option[ReferrerUrl]
   ): Action[AnyContent] =
-    Action.async { implicit request =>
+    Action.asyncUsing { request ?=>
       Future.successful {
```

Comment:
As per the other comment, my preference is for:
```
Action.async { request =>
    given Request[AnyContent] = request
   Future.successful { ... 
```

Reply:
updated now